zp-relayer/workers/sentTxWorker.ts

```diff
@@ -110,6 +117,9 @@ export async function createSentTxWorker<T extends EstimationType>(gasPrice: Gas
         pool.optimisticState.rollbackTo(pool.state)
         logger.info('Clearing optimistic nullifiers...')
         await pool.optimisticState.nullifiers.clear()
+        logger.info('Clearing optimistic roots...')
+        await pool.optimisticState.roots.clear()
```

Comment:
Is there a way to aggregate several operations to a single "transaction" ?
There seems to be some risk for consistency if some changes apply, while others don't

Reply:
>Is there a way to aggregate several operations to a single "transaction" ?

Don't get it. Could you elaborate?

Reply:
Yes, there are atomic transactions in redis. Will implement it later, as this change will cause merge conflicts for https://github.com/zkBob/zeropool-relayer/pull/78 .
Created an issue for this https://github.com/zkBob/zeropool-relayer/issues/82